tools/generate-release-artifacts.sh

```diff
@@ -36,19 +43,22 @@ function step_complete() {
 }
 
 function build_binary_arm64() {
-  build_binary "arm64"
+  build_binary "linux" "arm64"
+  build_binary "darwin" "arm64"
 }
 
 function build_binary_amd64() {
-  build_binary "amd64"
+  build_binary "linux" "amd64"
+  build_binary "darwin" "amd64"
```

Comment:
we don't use this script to generate darwin binaries, this script is only used when the CI pipeline that generates release binaries (linux/windows) fails. We have a separate script that generates signed mac binaries, so I would remove this line

Reply:
Done.